src/handle_action.cpp

```diff
@@ -1334,23 +1334,20 @@ static void open_movement_mode_menu()
 
     as_m.text = _( "Change to which movement mode?" );
 
-    as_m.entries.emplace_back( 0, true, 'w', _( "Walk" ) );
-    as_m.entries.emplace_back( 1, true, 'r', _( "Run" ) );
-    as_m.entries.emplace_back( 2, true, 'c', _( "Crouch" ) );
+    as_m.entries.emplace_back( CMM_RUN, true, 'r', _( "Run" ) );
+    as_m.entries.emplace_back( CMM_WALK, true, 'w', _( "Walk" ) );
+    as_m.entries.emplace_back( CMM_CROUCH, true, 'c', _( "Crouch" ) );
+    as_m.entries.emplace_back( CMM_COUNT, true, '"', _( "Cycle move mode (run/walk/crouch)" ) );
```

Comment:
I need to verify that this won't generate a new translatable string, it *shouldn't*, but if it does it needs to go.

Reply:
Hopefully it won't, I selected that string because it already appeared in translation files.